DemocracyClub/Website
# Hotfix/blog tags

Closes https://github.com/DemocracyClub/Website/issues/734, https://github.com/DemocracyClub/Website/issues/750, https://github.com/DemocracyClub/Website/issues/711

https://stage.democracyclub.org.uk/blog/

This work also:

- Adds featured posts to the home page context (although not currently used on the home page)
- Renders a menu of blog tags from published posts on the blog post list page so the user can choose and filter
- Includes misc text changes

To test: 
1. https://github.com/DemocracyClub/Website/issues/734: Add alt text to a hero image in the admin panel. Then navigate to the blog and check the source code of the hero image to confirm presence of the same alt text. 
2. https://github.com/DemocracyClub/Website/issues/750 I've added instructions to do this on README.md.
3. https://github.com/DemocracyClub/Website/issues/711 Create a new blog and add the title. A slug should be auto generated from the title. 

![Screenshot 2023-10-18 at 4 51 55 PM](https://github.com/DemocracyClub/Website/assets/7017118/f850f47d-24ef-4dc9-ab41-f4e853264f51)

![Screenshot 2023-10-19 at 9 19 52 AM](https://github.com/DemocracyClub/Website/assets/7017118/c79997dc-d7ff-4eeb-b48e-4cdc7f2db779)



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205727091804968
